src/tox/util/pep517/backend.py

```diff
@@ -1,4 +1,7 @@
-"""Handles communication on the backend side between frontend and backend"""
+"""Handles communication on the backend side between frontend and backend
+
+PLEASE KEEP THIS FILE PYTHON 2.7 COMPATIBLE
```

Comment:
There is no need to use all caps. Please add a reference to the development documentation so you can read about the reasons.